File: PinchZoom/app/src/main/java/de/afarber/pinchzoom/MainActivity.java
package de.afarber.pinchzoom;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {
    protected final static String TAG = "PinchZoom";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: MagicApp/app/src/main/java/de/afarber/magicapp/MainActivity.kt
package de.afarber.magicapp

import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.Surface
import androidx.compose.ui.Modifier
import de.afarber.magicapp.ui.app.magicAppRoot
import de.afarber.magicapp.ui.theme.magicAppTheme

class MainActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            magicAppTheme {
                Surface(
                    modifier = Modifier.fillMaxSize(),
                    color = MaterialTheme.colorScheme.background,
                ) {
                    magicAppRoot()
                }
            }
        }
    }
}
